### PR TITLE
Add ebib-jump-to-entry

### PR DESCRIPTION
**Pull request by Vonfry**

### CODE REVIEW

File: modes/ebib/evil-collection-ebib.el [hunk 1]
```diff
@@ -47,6 +47,7 @@
     "k" 'ebib-prev-entry
     "n" 'ebib-search-next
     "p" 'ebib-yank-entry
+    "gj" 'ebib-jump-to-entry
```
Comment (condy0919) [inline modes/ebib/evil-collection-ebib.el:50]: IMHO, `J` is better than `gj`.

**Review by condy0919 — CHANGES_REQUESTED** [review]

File: modes/ebib/evil-collection-ebib.el (hunk 1 above)
Comment (Vonfry) [reply modes/ebib/evil-collection-ebib.el:50]: `J` has been used by ebib for jump to other database.

Comment (condy0919) [reply modes/ebib/evil-collection-ebib.el:50]: There is no `J` binding defined in `ebib-index-mode-map` and `ebib-entry-mode-map`.

Comment (Vonfry) [reply modes/ebib/evil-collection-ebib.el:50]: [https://github.com/joostkremers/ebib/blob/37b9c4ec7a57ffdda53f3345cc99cf6819e94863/ebib.el#L966](source)

Comment (condy0919) [reply modes/ebib/evil-collection-ebib.el:50]: OK

**Pull request merged**